concepticondata/conceptlists/Girard-1971-546.tsv

```diff
@@ -0,0 +1,547 @@
+Girard-1971-546-450	450	jacuú	jacuú			415	*jacuú
+Girard-1971-546-451	451	cerrar	cerrar	1046	SHUT	416	*cerrar
+Girard-1971-546-452	452	esconderse	esconderse	2485	HIDE (ONESELF)	417	*esconderse
+Girard-1971-546-453	453	chaco	chaco			418	*chaco
+Girard-1971-546-454	454	fuego	fuego	221	FIRE	419	*fuego
+Girard-1971-546-455	455	apagar	apagar	127	EXTINGUISH	419	*apagar
+Girard-1971-546-456	456	quemar	quemar	141	BURN (SOMETHING)	419	*quemar
+Girard-1971-546-457	457	tronco,pierna	tronco,pierna			420	*tronco,pierna
+Girard-1971-546-458	458	pierna,falda	pierna,falda			420	*pierna,falda
+Girard-1971-546-459	459	pierna,muslo	pierna,muslo			420	*pierna,muslo
+Girard-1971-546-460	460	dar	dar	1447	GIVE	421	*dar
+Girard-1971-546-461	461	detrás,atrás	detrás,atrás	1686	BEHIND	421	*detrás,atrás
+Girard-1971-546-462	462	atrás,después	atrás,después			421	*atrás,después
+Girard-1971-546-463	463	abajo	abajo	1326	DOWN	421	*abajo
+Girard-1971-546-464	464	reflexive suffix	reflexive suffix			422	*reflexive suffix
+Girard-1971-546-465	465	uña	uña	1896	FINGERNAIL OR TOENAIL	423	*uña
+Girard-1971-546-466	466	cintura	cintura	1681	WAIST	424	*cintura
+Girard-1971-546-467	467	centro	centro	491	CENTER	425	*centro
+Girard-1971-546-468	468	raíz	raíz	670	ROOT	426	*raíz
+Girard-1971-546-469	469	terminar	terminar	1766	FINISH	427	*terminar
+Girard-1971-546-470	470	tronar	tronar			428	*tronar
+Girard-1971-546-471	471	quejido	quejido			429	*quejido
+Girard-1971-546-472	472	pisar,arañar	pisar,arañar			430	*pisar,arañar
+Girard-1971-546-473	473	3ra person	3ra person			431	*3ra persona
+Girard-1971-546-474	474	barriga	barriga	1251	BELLY	432	*barriga
+Girard-1971-546-475	475	encender	encender	737	LIGHT (IGNITE)	433	*encender
+Girard-1971-546-476	476	cortar,trozar	cortar,trozar			434	*cortar,trozar
+Girard-1971-546-477	477	loro (chico)	loro (chico)	882	PARROT	435	*loro (chico)
+Girard-1971-546-478	478	ojo	ojo	1248	EYE	436	*ojo
+Girard-1971-546-479	479	motacú	motacú			437	*motacú
+Girard-1971-546-480	480	piedra	piedra	857	STONE	438	*piedra
+Girard-1971-546-481	481	lombriz	lombriz	1219	WORM	439	*lombriz
+Girard-1971-546-482	482	pesar	pesar	1098	WEIGH	440	*pesar
+Girard-1971-546-483	483	nieta,nieto	nieta,nieto	1618	GRANDSON	441	*nieta,nieto
+Girard-1971-546-484	484	sobrino,sobrina	sobrino,sobrina	1063	NEPHEW	441	*sobrino,sobrina
+Girard-1971-546-485	485	hermano mayor	hermano mayor	1759	OLDER BROTHER	441	*hermano mayor
+Girard-1971-546-486	486	hermano	hermano	1262	BROTHER	441	*hermano
+Girard-1971-546-487	487	así	así			442	*así
+Girard-1971-546-488	488	espuma	espuma	1768	FOAM	443	*espuma
+Girard-1971-546-489	489	luz	luz	630	LIGHT (RADIATION)	444	*luz
+Girard-1971-546-490	490	silbar	silbar	1025	WHISTLE	445	*silbar
+Girard-1971-546-491	491	tabaco	tabaco	974	TOBACCO	446	*tabaco
+Girard-1971-546-492	492	marfil (palmera)	marfil (palmera)			447	*marfil (palmera)
+Girard-1971-546-493	493	caerse	caerse	2894	TUMBLE (FALL DOWN)	448	*caerse
+Girard-1971-546-494	494	sardina	sardina			449	*sardina
```

Comment:
Possibly a new concept we can add, there are already two other cases. Please open an issue based on there cases of "new-concept-proposals". We will then discuss whether this concept should indeed be added, or not